grype/match/ignore_test.go

```diff
@@ -70,19 +70,22 @@ var (
 		{
 			Vulnerability: vulnerability.Vulnerability{
 				ID:        "CVE-458",
-				Namespace: "ruby-vulns",
+				Namespace: "java-vulns",
```

Comment:
Has the change to this vulnerability test removed the ruby case? Unclear why it should be removed, should this be added back in as another test case?